frontend/src/customAxios.ts

```diff
@@ -12,8 +12,8 @@ const defaultEnvelope = (x: any) => ({ data: { records: x } });
  * errorToast is displayed when the request fails for any reason. By default this will return an error from axios.
  */
 export interface LifecycleToasts {
-  loadingToast: () => React.ReactText;
-  successToast: () => React.ReactText;
+  loadingToast?: () => React.ReactText;
```

Comment:
allow apis to skip lifecycle notifications where relevant.

Reply:
I like this